pkg/cmd/server/api/v1/types.go

```diff
@@ -970,8 +970,16 @@ type KubernetesMasterConfig struct {
 	ServicesNodePortRange string `json:"servicesNodePortRange"`
 	// StaticNodeNames is the list of nodes that are statically known
 	StaticNodeNames []string `json:"staticNodeNames"`
+
 	// SchedulerConfigFile points to a file that describes how to set up the scheduler. If empty, you get the default scheduling rules.
 	SchedulerConfigFile string `json:"schedulerConfigFile"`
+	// SchedulerName is the name this scheduler uses to assign pods.
```

Comment:
I'm sort of surprised these options weren't already part of the scheduler config object read from the file